dataproc_jupyter_plugin/handlers.py

```diff
@@ -119,6 +120,10 @@ def get_cached_credentials():
 
         return credentials
 
+def gcp_service_url(service_name, default_url=None):
+    default_url = default_url or f'https://{service_name}.googleapis.com/'
+    configured_url = get_gcloud_config(f'configuration.properties.api_endpoint_overrides.{service_name}')
+    return configured_url or default_url
```

Comment:
There's a missing blank line between the method definitions, but I may merge this and then fix it in a follow up.